src/components/MenuBar.vue

```diff
@@ -1,6 +1,6 @@
 <script setup>
 import ocLogo from "/src/oc-logo-maroon.png";
-import { computed, ref, onMounted } from "vue";
+import { computed, ref, onMounted, nextTick } from "vue";
```

Comment:
this and console.logs should be removed

Reply:
There are several more

Reply:
Since I need his changes so I can work on the next story, I will merge his branch and delete the console.logs on my branch